db/migrate/20240326171810_add_fcc_import_to_project_models.rb

```diff
@@ -0,0 +1,5 @@
+class AddFccImportToProjectModels < ActiveRecord::Migration[7.0]
+  def change
+    add_column :core_data_connector_project_models, :allow_fcc_import, :boolean
```

Comment:
I might be missing something, but it looks like this column isn't actually used anywhere?

My assumption is that only `items` records that belong to `project_models` with this value set to "true" would execute the `after_create` callback? I think it makes sense to have some sort of assertion like this, but it might also be worth considering whether this should live on the `project_models` table. It's presence on the `project_models` table implies that we could allow importing from FairCopy Cloud for multiple models (of any type) within a project. Where more likely, we'll only ever want to import from FairCopy Cloud for a _single_ model of type "Item" per project.

Would it make more sense to have a `fair_copy_cloud_project_model_id` (or something similar) column on the `projects` table?